### PR TITLE
Update license files to comply with Apache 2.0

### PR DESCRIPTION
**Pull request by rtpro**
Depends on PR #119

### CODE REVIEW

**Comment by rtpro** [comment]
@isaac-io Please review this PR.

**Review by isaac-io — APPROVED** [review]

File: AUTHORS [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Speedb
```
Comment (isaac-io) [inline AUTHORS:1]: This is not a Markdown file. I think we can just remove this line.

**Review by isaac-io — CHANGES_REQUESTED** [review]

**Comment by rtpro** [comment]
Depends on PR #103 
We need to remove GPL based dependency gnu_parallel - PR #103

**Comment by isaac-io** [comment]
I opened #119 for the removal of the GPL.

**Pull request merged**